manager/apis/v2/handler/cdn_cluster.go

```diff
@@ -28,25 +26,21 @@ import (
 func (handler *Handler) CreateCDNCluster(ctx *gin.Context) {
 	var cluster types.CDNCluster
 	if err := ctx.ShouldBindJSON(&cluster); err != nil {
-		NewError(ctx, http.StatusBadRequest, err)
+		ctx.Error(NewError(http.StatusBadRequest, err))
```

Comment:
1.  ctx.Error just append a new error，NewError serializes the given struct as JSON into the response body and reply to http request.  they have different functions， and not equa.

Reply:
@yangjun289519474 I  already change The NewError func. And I add a new handler.Error midware.